.github/workflows/mindsdb.yml

```diff
@@ -67,6 +67,9 @@ jobs:
           cat mindsdb/interfaces/storage/db.py | grep integration
 
           env PYTHONPATH=./ pytest tests/unit/test_executor.py
+        
+          # need fix in LW. not working yet 
```

Comment:
test is commented because it brakes CI. it works well locally
it will be commented till this issue won't solved
https://github.com/mindsdb/lightwood/issues/1036

Reply:
also added commented test_lightwood.py, to not forget it